contrib/bgsegm_test.go

```diff
@@ -0,0 +1,28 @@
+package contrib
+
+import (
+	"testing"
+	"gocv.io/x/gocv"
+	v "gocv.io/x/gocv/contrib"
```

Comment:
The test fails without this line. I've no idea why.

Reply:
why "v" as package name and not the default "contrib"?

Reply:
you also need to gofmt your changes.

Reply:
I pulled your branch, merged the current `dev` branch and removed the line importing `gocv.io/x/gocv/contrib` (i believe it had some circular imports). Seems like the tests pass now without it.